service_crategen/src/commands/generate/codegen/rest_json.rs

```diff
@@ -77,7 +77,11 @@ impl GenerateProtocol for RestJsonGenerator {
 
                     request.sign(&self.credentials_provider.credentials()?);
 
-                    let response = self.dispatcher.dispatch(&request)?;
+                    let mut result = try!(self.dispatcher.dispatch(&request));
```

Comment:
Yup that did the trick!  Weird how we have seven services that were affected by that change but none of their tests failed.  We should level up our integration tests!

I removed three lines:

```rust
-                    let mut result = try!(self.dispatcher.dispatch(&request));
-                    let mut body: Vec<u8> = Vec::new();
-                    try!(result.body.read_to_end(&mut body));
```